contracts/contracts/core/PreconfManager.sol

```diff
@@ -400,19 +400,19 @@ contract PreconfManager is
 
     /**
      * @dev Initiate a slash for a commitment.
-     * @param commitmentIndex The hash of the commitment to be slashed.
+     * @param commitmentDigest The hash of the commitment to be slashed.
      * @param residualBidPercentAfterDecay The residual bid percent after decay.
      */
     function initiateSlash(
-        bytes32 commitmentIndex,
+        bytes32 commitmentDigest,
         uint256 residualBidPercentAfterDecay
     ) public onlyOracleContract whenNotPaused {
         OpenedCommitment storage commitment = openedCommitments[
```

Comment:
This mapping also uses commitmentIndex

Reply:
We should change the name in the mapping as well, they're the same value, I just think we should keep a consistent variable name